life_expectancy/cleaning.py

```diff
@@ -0,0 +1,136 @@
+"""Module for cleaning life expectancy data."""
+import argparse
+import pandas as pd
+
+def main(region: str) -> None:
+    """Main function."""
+    full_data = _load_data('life_expectancy/data')
+    region_data = _clean_data(full_data, region)
+    _save_data_csv('life_expectancy/data', region_data, region)
+
+def _clean_data(data: pd.DataFrame, region: str) -> pd.DataFrame:
+    """ Clean the data and return only a given region.
+
+    Args:
+        region: the name of the region to clean
+    """
+    full_data = (
+        data
+        .pipe(_melt_table)
+        .pipe(_clean_year_value_cols)
+        .pipe(_clean_region_col)
```

Comment:
is there an advantage of doing these 3 functions with .pipe() instead of creating, for example, a preprocessing function that calls all of them? I'm thinking in terms of scalability, but I'm not sure which would be a better approach

Reply:
Don't know what is the best approach really, it is a good question. Lets discuss this in the meeting, but yes it could be used in a preprocessing function.